tests/get_or_create/tests.py

```diff
@@ -466,6 +467,30 @@ def raise_exception():
         )
         self.assertFalse(created)
 
+    def test_update_only_defaults(self):
```

Comment:
A test for the MTI case that uses non-local concrete fields would be great as well.